src/components/CreateResolution/CompleteResolution.vue

```diff
@@ -516,7 +516,7 @@ export default class CompleteResolution extends Mixins(CommonMixin, DateMixin, E
   }
 
   private get isConfirmResolutionValid (): boolean {
-    return this.$refs.confirmResolutionChkRef.valid
+    return this.$refs.confirmResolutionChkRef?.valid
```

Comment:
When resuming from payment or save failures, Step 2 was seeing a type error which prevent the Checkbox component from re rendering. This and the optional chains below fix that. 
I believe we saw this bug before in other instances, not sure if we had a bug ticket for this, but this will fix it.